8.x/theme-development/client-side-cart-module.md

```diff
@@ -0,0 +1,52 @@
+## Client Side cart module
+
+[[toc]]
+
+The Larammerce Template Engine has the **local card module**. In this tutorial, the source code of this module is Reviewed.
+
+**local card module** is in the following directory `resources/assets/js/define/local_cart_service.js`.
+It has 437 lines of code and is based on require js but there is no requirement to use require js. To read more about it, refer to this address:[requirejs](https://www.requirejs.org/).
+
+Let's check the local card module:
+
+requirejs has two very important functions called define and require. In this module, there is a define function that takes three inputs:
```

Comment:
Put the `define` and `require` inside a couple of graves.

Reply:
As for the `define` function, which is used in the first line of this file, it takes three parameters as input parameters. The first input is the name of the module; The second input is an ArrayList of the dependencies of this module
and the third input is the body function of the module.